locale/de-DE.ini

```diff
@@ -0,0 +1,11 @@
+VirtualCam="Virtuelle Kamera"
+DelayFrame="Gepufferte Einzelbilder"
+StartOutput="Start"
+StopOutput="Stopp"
+AutoStart="Automatischer Start"
+Reset="Zurücksetzen"
+Option="Einstellungen"
+HorizontalFlip="Horizontal flippen"
```

Comment:
Here I would use "Horizontal spiegeln" instead of "flippen"

Reply:
I agree, that's better.